examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -90,14 +87,15 @@ int main()
 		camera->OnAnimate(std::chrono::duration_cast<std::chrono::milliseconds>(device->getTimer()->getTime()).count());
 		camera->render();
 
+        asset::SBasicViewParameters uboData;
 		core::matrix4SIMD mvp = camera->getConcatenatedMatrix();
-        memcpy(uboData.data(), mvp.pointer(), sizeof(core::matrix4SIMD));//MVP
+        memcpy(uboData.MVP, mvp.pointer(), 4*4*4);
```

Comment:
please for the love of god don't use numerical literals as stand-ins for sizes, always `sizeof` and of the correct struct type!